### PR TITLE
small cleanup in  sexp.cpp

### PR DESCRIPTION
**Pull request by CasparKielwein**
I looked for possibilities to reduce the size of the parse/sexp.cpp file.
I did not fine any obvious code duplication, so here is a small PR with changes to reduce the amount of warnings from clang-tidy.

Best reviewed on a per commit basis.
The last commit contains only white-space changes which replaces all occurrences of 4 white-space with a tab, to make the indentation consistent within the file.

### CODE REVIEW

File: code/parse/sexp.cpp [hunk 1]
```diff
@@ -17145,20 +17142,6 @@ void sexp_set_countermeasures(int node)
...
-
-	Current_sexp_network_packet.get_ship(shipp);
-	if (shipp == NULL) {
-		return;
-	}
-	if (Current_sexp_network_packet.get_int(num_cmeasures)) {
-		shipp->cmeasure_count = num_cmeasures;
-	}
-}
-
```
Comment (Goober5000) [inline code/parse/sexp.cpp:17161]: This function should not be deleted.  Instead, `multi_sexp_set_countermeasures` should be added amidst the rest of the multi sexp calls.

File: code/parse/sexp.cpp [hunk 2]
```diff
@@ -21817,14 +21799,6 @@ void sexp_reset_time_compression()
 	 Current_sexp_network_packet.end_callback();
 }
 
-void multi_sexp_reset_time_compression() 
-{
-	set_time_compression(1);
-	lock_time_compression(false);
-}
```
Comment (Goober5000) [inline code/parse/sexp.cpp:21824]: This should also be added to the multi sexp list.

File: code/parse/sexp.cpp (hunk 1 above)
Comment (CasparKielwein) [reply code/parse/sexp.cpp:17161]: can you give me a hint what i need to do to add it to the appropriate list?

Comment (Goober5000) [reply code/parse/sexp.cpp:17161]: I thought it wouldn't be hard to figure out, but sure.  Look in `multi_sexp_eval` where all the other multi_sexp_* functions are called.  Add the appropriate `case` statements and function calls to the big `switch` block.

**Comment by CasparKielwein** [comment]
Failure of the appveyor build seems unrelated.
It is caused by one run of the compiler running out of memory.

**Comment by Goober5000** [comment]
Ok, I restarted it.

**Pull request merged**